server/public/styles.css

```diff
@@ -0,0 +1,173 @@
+.hidden {
+  display: none !important;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+html, body {
+  min-height: 680px;
+  max-width: 900px;
+  margin: 0;
+  padding: 0;
```

Comment:
Bootstrap has spacing helpers you should use for margin and padding in most situations.

https://getbootstrap.com/docs/4.5/utilities/spacing/